pkg/gui/controllers/helpers/window_arrangement_helper.go

```diff
@@ -34,14 +34,14 @@ func NewWindowArrangementHelper(
 
 const INFO_SECTION_PADDING = " "
 
-func (self *WindowArrangementHelper) GetWindowDimensions(informationStr string, appStatus string) map[string]boxlayout.Dimensions {
+func (self *WindowArrangementHelper) GetWindowDimensions(informationStr string, appStatus string, forcePortraitMode bool) map[string]boxlayout.Dimensions {
```

Comment:
The UserConfig struct is available to WindowArrangementHelper as `self.c.UserConfig`, so you don't have to pass it in.